lib/adiwg/mdtranslator/readers/iso19115_3/modules/module_identification.rb

```diff
@@ -23,11 +23,11 @@ def self.unpack(xMetadata, _hResponseObj)
                   id = xMetadata.xpath(ADIWG::Mdtranslator::Readers::Iso191153::CODE_XPATH)
                   cs = xMetadata.xpath(ADIWG::Mdtranslator::Readers::Iso191153::CODESPACE_XPATH)
                   desc = xMetadata.xpath(ADIWG::Mdtranslator::Readers::Iso191153::DESC_XPATH)
-                  xMetadata.xpath(ADIWG::Mdtranslator::Readers::Iso191153::TITLE_XPATH)
+                  # version = xMetadata.xpath(ADIWG::Mdtranslator::Readers::Iso191153::VERSION_XPATH)
 
                   hIdentifier[:identifier] = id[0].text
                   hIdentifier[:namespace] = cs[0].text
-                  hIdentifier[:version] = nil # TODO
+                  # hIdentifier[:version] = version[0].text # TODO
```

Comment:
Can we have more detail here about why these were changed AND commented out?

Reply:
i decided to put `version` on hold because it requires more thought and I wanted to move onto something i could immediately solve. commenting it out isn't necessary. i'll work on this next.